src/main/scala/no/gdl/bookapi/BookApiProperties.scala

```diff
@@ -20,6 +20,7 @@ import scala.util.{Failure, Success}
 object BookApiProperties extends LazyLogging {
   val RoleWithWriteAccess = "books:write"
   val SecretsFile = "book-api.secrets"
+  val FeaturedContentAdminRole = "book-api:featuredContentAdmin"
```

Comment:
Hmm...kanskje prematurt å tenke på hva rollene skal hete/dekke, men ser for meg at vi trolig ikke trenger så spesifikke roller som `featuredContentAdmin`. Kanskje kun `admin` holder? Åpen for innspill